cmd/logout.go

```diff
@@ -0,0 +1,81 @@
+// Copyright © 2016 Dropbox, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+)
+
+const revokeURL = "https://api.dropboxapi.com/2/auth/token/revoke"
+
+// Sends a POST request to /auth/token/revoke to revoke the given API token.
+func revoke(tok string) error {
```

Comment:
The SDK already has a `auth.TokenRevoke` API that you can use. See `initDbx` in `root.go` on how to use `options.Domain` to get a `Client` object. Once you have that, you can simply call `TokenRevoke` on the object for each registered domain.